packages/node/README.md

```diff
@@ -39,66 +39,63 @@ const DashX = require('@dashx/node');
 const dashx = DashX.createClient();
 ```
 
-## Examples
-
-#### Multiple recipients
+### Deliver
 
 ```javascript
-dashx.deliver('message-identifier', {
+dashx.deliver({
   to: 'John Doe <john@example.com>',
-  cc: ['admin@example.com', 'sales@example.com>'],
+  body: 'Hello World!'
 });
 ```
 
-#### Template variables
+`deliver` can accept multiple recipients like so:
 
 ```javascript
-dashx.deliver('message-identifier', {
-  to: 'john@example.com',
-  data: { name: 'John' },
+dashx.deliver({
+  to: ['John Doe <john@example.com>','admin@example.com', 'sales@example.com>'],
+  body: 'Hello World!'
 });
 ```
 
-#### Attachment support
+### Identify
+
+- Existing User
 
 ```javascript
-dashx.deliver('message-identifier', {
-  to: 'jane@example.com',
-  attachments: [
-    {
-      file: '/path/to/handbook.pdf',
-      name: 'Handbook',
-    },
-  ],
+dashx.identify('uid_of_user', {
+  firstName: 'John',
+  lastName: 'Doe',
+  email: 'john@example.com',
+  phone: '+1-234-567-8910'
 });
 ```
 
-#### Use as `promise`
+This will override details of an existing user for the provided `uid`.
 
-`dashx.deliver` returns a promise so you can chain other tasks after successfully sending mail.
+- New User
```

Comment:
This is actually the use case for anonymous user. I think the instructions make it seem like new user (signup) vs existing user (login). However, even in the signup case, the developer would have to create the user first, and then create the profile using the UID produced within their system.